src/containers/ConfirmPassword/index.js

```diff
@@ -1,3 +1,50 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { compose } from 'recompose';
+import { withAuth } from '../Session/index';
+import { withFirebase } from '../Firebase';
 import ConfirmPassword from './confirmPass';
 
-export default ConfirmPassword;
+class Index extends Component {
+  state = {
+    errorMessage: '',
+  };
+
+  handleChange = () => {
```

Comment:
Why do we need this function?

Reply:
to empty the error message when the user writes on the input 
--> the error msg comes from the firebase 
@yosefalnajjarofficial